cmd/credential.go

```diff
@@ -116,21 +112,21 @@ func createCredential(accessKey, secretKey string) (cred credential, err error)
 }
 
 // Initialize a new credential object
-func mustGetNewCredential() credential {
+func mustGetNewCredential(accessKeyLen, secretKeyLen int) credential {
```

Comment:
I see this signature is used only in jwt_test.go.  You can leave the current signature and create another function like
```
func GetNewCredential(accessKeyLen, secretKeyLen int) (credential, error) {
// same as this mustGetNewCredential() except returning credential and error
}
```

then, you could have `mustGetNewCredential()` like
```
func mustGetNewCredential() credential {
    cred, err := GetNewCredential(accessKeyMaxLen, secretKeyMaxLen)
    fatalIf(err, "Unable to generate new credential.")
    return cred
}
```

In jwt_test.go, you could use `GetNewCredential()` than `mustGetNewCredential()`